pyproject.toml

```
@@ -147,7 +147,8 @@ docstring-quotes = "double"
 inline-quotes = "single"
 
 [tool.ruff.lint.isort]
-known-first-party = ["apify", "apify_client", "apify_shared"]
+known-first-party = ["apify_client", "apify_shared"]
```

Could you please remove `known-first-party` completely and leave there only `known-local-folder = ["apify"]`?
 
```suggestion
known-local-folder = ["apify"]
```

We agreed with @janbuchar to not to create a new section in our imports, so `apify-shared` and `apify-client` will be part of standard 3rd party imports.